test/Models/WebMapServiceCatalogItemSpec.js

```diff
@@ -472,6 +486,15 @@ describe("WebMapServiceCatalogItem", function() {
       .load()
       .then(function() {
         expect(wmsItem.intervals.length).toEqual(1);
+
+        const firstInterval = wmsItem.intervals.get(0);
+        // expect(firstInterval.data).toEqual("2015-04-27T06:15:00.000Z");
+        // expect(typeof firstInterval.data).toEqual("string");
```

Comment:
Yeah this is a weird case. Re-reading Annex D in the WMS spec, I think it might not even be valid. It would have to mean "any time in this interval is acceptable, and consider the layer to be continuously varying over that interval". I'm not really sure what we could do with that, and it's hard to even imagine a scenario where a server would really have such a thing. I guess if it were able to interpolate values at any time it might offer a time dimension like this? In any case, unless you come across an actual server like this, I'm happy to put it aside / not worry about it.

Reply:
Whether there is a WMS server out there that would do it or not is one thin, it may well be dead code... That said if there is a server out there we probably shouldn't be sending a moment object but a datestring.

I suggest we leave in the handling for the timebeing, I'll fix the handling in #3352 so that we have working code, and then we can kill it off as a next step. THat way we've at least got something to revert back to just in case...